servers/curated-corpus-api/src/admin/resolvers/queries/Section/getSectionWithSectionItems.integration.ts

```diff
@@ -232,4 +232,151 @@ describe('queries: Section (getSectionsWithSectionItems)', () => {
     expect(result.body.errors).toBeUndefined();
     expect(result.body.data?.getSectionsWithSectionItems).toEqual([]);
   });
+
+  it('should filter sections by createSource when provided', async () => {
+    // Create a MANUAL section
+    await createSectionHelper(db, {
+      externalId: 'manual-section-1',
+      createSource: ActivitySource.MANUAL,
+      scheduledSurfaceGuid: ScheduledSurfacesEnum.NEW_TAB_EN_US,
+      active: true,
+      disabled: false,
+    });
+
+    // Query for only ML sections
+    const mlResult = await request(app)
+      .post(graphQLUrl)
+      .set(headers)
+      .send({
+        query: print(GET_SECTIONS_WITH_SECTION_ITEMS),
+        variables: {
+          scheduledSurfaceGuid: "NEW_TAB_EN_US",
+          createSource: "ML"
+        },
+      });
+
+    expect(mlResult.body.errors).toBeUndefined();
+    // Should return only ML sections (3 from beforeEach)
+    expect(mlResult.body.data?.getSectionsWithSectionItems.length).toEqual(3);
+    expect(mlResult.body.data?.getSectionsWithSectionItems.every(s => s.createSource === 'ML')).toBeTruthy();
+
+    // Query for only MANUAL sections
+    const manualResult = await request(app)
+      .post(graphQLUrl)
+      .set(headers)
+      .send({
+        query: print(GET_SECTIONS_WITH_SECTION_ITEMS),
+        variables: {
+          scheduledSurfaceGuid: "NEW_TAB_EN_US",
+          createSource: "MANUAL"
+        },
+      });
+
+    expect(manualResult.body.errors).toBeUndefined();
+    // Should return only MANUAL section
+    expect(manualResult.body.data?.getSectionsWithSectionItems.length).toEqual(1);
+    expect(manualResult.body.data?.getSectionsWithSectionItems[0].externalId).toEqual('manual-section-1');
+    expect(manualResult.body.data?.getSectionsWithSectionItems[0].createSource).toEqual('MANUAL');
+  });
+
+  it('should compute section status correctly', async () => {
```

Comment:
with the very good unit tests on the `computeSectionStatus` function, i think we can drop this test.